scripts/templates/grafana-dashboards/dm-apps.json

```diff
@@ -0,0 +1,747 @@
+{
+  "title": "DM-Apps",
+  "tags": [],
+  "style": "dark",
+  "timezone": "browser",
+  "editable": true,
```

Comment:
If we set this to `false` it removes the Edit buttons and prevents the dashboards from being modified in the interface, so I think we should do that.